anybox/recipe/odoo/base.py

```diff
@@ -891,7 +890,7 @@ def retrieve_addons(self):
             if subdir:
                 addons_dir = join(addons_dir, subdir)
 
-            manifest = os.path.join(addons_dir, '__openerp__.py')
+            manifest = os.path.join(addons_dir, '__odoo__.py')
```

Comment:
I guess we should manage that ``__openerp__.py`` or ``__manifest__.py`` not ``__odoo__.py``